apps/studio/components/interfaces/Settings/Database/DatabaseSettings/DatabaseConnectionString.tsx

```diff
@@ -65,162 +96,111 @@ const DatabaseConnectionString = () => {
-
-                <Tabs.Panel id="nodejs" label="Nodejs">
-                  <Input
-                    copy
-                    readOnly
-                    disabled
-                    value={uriConnString}
-                    onCopy={() => {
-                      handleCopy('Nodejs')
-                    }}
-                  />
-                </Tabs.Panel>
-
-                <Tabs.Panel id="php" label="PHP">
-                  <Input
-                    copy
-                    readOnly
-                    disabled
-                    value={golangConnString}
-                    onCopy={() => {
-                      handleCopy('PHP')
-                    }}
-                  />
-                </Tabs.Panel>
-
-                <Tabs.Panel id="python" label="Python">
-                  <Input
-                    copy
-                    readOnly
-                    disabled
-                    value={
-                      `user=${connectionInfo.db_user} password=[YOUR-PASSWORD]` +
-                      ` host=${connectionInfo.db_host} port=${connectionInfo.db_port.toString()}` +
-                      ` database=${connectionInfo.db_name}`
-                    }
-                    onCopy={() => {
-                      handleCopy('Python')
-                    }}
-                  />
-                </Tabs.Panel>
-              </Tabs>
-            )}
-          </Panel.Content>
-        </Panel>
-      </section>
+            <Tabs
+              type="underlined"
+              size="tiny"
+              activeId={selectedTab}
+              baseClassNames="!space-y-0 -mb-[1px]"
+              onChange={setSelectedTab}
+            >
+              {CONNECTION_TYPES.map((type) => (
+                <Tabs.Panel key={type.id} id={type.id} label={type.label} />
+              ))}
+            </Tabs>
+          </div>
+        }
+      >
+        <Panel.Content>
+          {isLoading && <ShimmeringLoader className="h-8 w-full" />}
+          {isError && <AlertError error={error} subject="Failed to retrieve database settings" />}
+          {isSuccess && (
+            <div className="flex flex-col gap-y-4">
+              <Alert_Shadcn_ variant="warning">
+                <IconAlertTriangle strokeWidth={2} />
+                <AlertTitle_Shadcn_>
+                  Direct database access via IPv4 and pgBouncer will be deprecated from January 26th
+                  2024
+                </AlertTitle_Shadcn_>
+                <AlertDescription_Shadcn_ className="space-y-3">
+                  <p>
+                    We strongly recommend using{' '}
+                    <a
+                      href="https://github.com/supabase/supavisor"
+                      target="_blank"
+                      rel="noreferrer"
+                      className="text-foreground underline underline-offset-[4px] decoration-brand-500 hover:decoration-foreground"
+                    >
+                      Supavisor
```

Comment:
instead of linking to Supavisor, we should say "we strongly recommend using connection pooling to connect to the database" and when they click on connection pooling it can take them to the previous section.

Reply:
![image](https://github.com/supabase/supabase/assets/19742402/265023ed-0a09-4b6e-961f-16b18c42652c)